src/main/java/com.springcore.lifecycle/Test2.java

```diff
@@ -12,7 +12,7 @@
 import com.springcore.collections.Emp;
 
 public class Test2 {
```

Comment:
Invalid naming conventions. use proper name to indicate the class's functionality